### DISCUSSION TITLE
AYUDA NO ENTIENDO

### DISCUSSION THREAD

**Discussion by marjosavi481** [atralice/Curso.Prep.Henry#128] [General]
Saludos tengo mucho problema al resolver ejercicios por cuenta propia, conozco los conceptos en general pero a la hora de resolver ejercicios no logro crear lo que me piden y la verdad me cuesta entender como resolver todo con la lógica que se necesita, es normal después de haber mirado mil veces todos los videos etc? siento que necesito un profesor.

**Comment by awesomnet1** [comment]
Suele ser normal al empezar que no se te ocurra una forma de resolver el problema, mi método para solucionarlo fue practicando y ver videos de gente resolviendo problemas.

**Comment by Andressan212** [comment]
a mi me pasa igual pero cuando practico  y anoto todo lo que yo hago  logro resolver y también ver video de como hacer cierto proyecto me ayuda

**Comment by carlosPadillaTorres** [comment]
¿Cómo te fue? ¿Si le entendiste?
